Aztec/Classes/TextKit/TextView.swift

```diff
@@ -242,12 +242,22 @@ open class TextView: UITextView {
             .font: defaultFont,
             .paragraphStyle: defaultParagraphStyle,
         ]
-        if let color = textColor {
+        if let color = defaultTextColor {
             attributes[.foregroundColor] = color
         }
         return attributes
     }
-    
+
+    open lazy var defaultTextColor: UIColor? = {
+        let color: UIColor
+        if #available(iOS 13.0, *) {
+            color = UIColor.label
```

Comment:
You could just return directly in each branch of this if statement, and you can drop the `UIColor`? (e.g. `return .label`)

Reply:
Done